Intermediate/algorithms/Selection Sort

```diff
@@ -0,0 +1,47 @@
+class SelectionSort 
```

Comment:
please change the file name to SelectionSort.java so that it is a compilable java file